### DISCUSSION TITLE
Show type of cargo and destination in vehicle list

### DISCUSSION THREAD

**Discussion by LubosKolouch** [OpenTTD/OpenTTD#11725] [Features]
## Version of OpenTTD
1.10.3

## Expected result
I can see what cargo the vehicle carries and where it goes. This is useful for stations with many vehicles coming and leaving.

## Actual result
List of vehicles that look the same, not knowing where it goes.
Look at the list - can you tell how many vehicles do you have carrying milk and where they are going?

## Steps to reproduce
1. Create a station
2. Assign many vehicles to the station
3. Try to find out which vehicle is which.

![image](https://user-images.githubusercontent.com/4145223/91578302-40aaab80-e94a-11ea-8761-781426cf34f0.png)

**Comment by valisak** [comment]
The train can be named at the time of purchase and will take over the name and order when cloning by further cloning.

**Comment by LubosKolouch** [comment]
> The train can be named at the time of purchase and will take over the name and order when cloning by further cloning.

This does not help, you still don't see in the station vehicles list the name...

**Comment by xordspar0** [comment]
The other vehicles already show the schedule and destination. Presumably trains don't show this information because trains are often long enough to cover up the spot where the schedule goes. So technically it might be easy to fix, but a UI needs to be designed for it.

![Gindington Transport, 2021-10-26](https://user-images.githubusercontent.com/2390950/147514327-b62699b6-3270-4158-aa68-a3d071fc10ab.png)

**Comment by 2TallTyler** [comment]
Partly solved by #8304 (shows cargo but not destination).
